pastepwn/analyzers/base64analyzer.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+
+class Base64Analyzer(RegexAnalyzer):
+    """Analyzer to match base64 encoding via regex"""
+    name = "Base64Analyzer"
+
+    def __init__(self, actions):
+        regex = r"^(?:[A-Za-z0-9+\/]{4})+(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$"
```

Comment:
I am not sure if matching the beginning and the end of the string is good. I know that it is the same regex as in my description, I just want to leave this here to leave a reminder.

Reply:
I've changed it to what you have provided and changed the first instance of `*` to `+` to match one or more to prevent regex matching empty strings.

The end of the string needs to be matched too as the end will change depending upon the padding. I will give it another look later tonight and maybe come up with a more robust solution, but this regex hasn't failed yet during testing and on quite large data too.